scalafix-core/shared/src/main/scala/scalafix/v1/Sym.scala

```diff
@@ -0,0 +1,149 @@
+package scalafix.v1
+
+import scala.meta.Tree
+import scala.meta.internal.{semanticdb3 => s}
+import scala.meta.internal.semanticdb3.SymbolInformation.{Property => p}
+import scala.meta.internal.semanticdb3.Scala._
+
+final class Sym private (val value: String) {
```

Comment:
Can we move `scala.meta.Symbol` to `v0` and rename this to `v1.Symbol`?

Reply:
My name is @olafurpg and I approve this message 👍